network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -343,7 +343,7 @@ class ProductPage(FoundationMetadataPageMixin, Page):
 
     @property
     def current_tally(self):
-        return sum(self.votes.get_votes())
+        return sum(self.votes.get_or_create_votes())
```

Comment:
Code in this file is the same as what's been reviewed in #5518 but I needed it here as well. Comes with the 0017 migration but will be renamed to 0018 when this PR is approved and rebased.